### PR TITLE
Symfonize planning/postes_cfg/lignes.php

### PR DESCRIPTION
**Pull request by sillydebs**

### CODE REVIEW

File: src/Controller/FrameworkController.php [hunk 1]
```diff
@@ -100,4 +100,88 @@ public function index (Request $request, Session $session){
...
+    public function addLine (Request $request, Session $session){
+        $CSRFToken = $GLOBALS['CSRFSession'];
+        $id = null;
+        $nom = null;
+
+        $this->templateParams(
+            array(
+                "CSRFToken"    => $CSRFToken,
+                "id"           => $id,
+                "nom"          => $nom
```
Comment (alexarn) [inline src/Controller/FrameworkController.php:116]: These 2 variables (id and nom) are useless here. When adding a new line, we never have id or nom

**Review by alexarn — CHANGES_REQUESTED** [review]
Encoding issue:
If you edit a line previously added (with legacy code) with accent, there an encoding issue.
I.e: "Rez de chaussée"

File: src/Controller/FrameworkController.php (hunk 1 above)
Comment (sillydebs) [reply src/Controller/FrameworkController.php:116]: They are required as parameters for the template, thus I have to declare them.

**Comment by alexarn** [comment]
Merged on master

**Pull request closed without merging**